package.json

```diff
@@ -11,5 +11,8 @@
     "debug": "^4.1.1",
     "http-errors": "^1.7.2",
     "selenium-webdriver": "4.0.0-alpha.1"
+  },
+  "devDependencies": {
+    "screener-runner": "^0.11.3"
```

Comment:
these dependencies should be in the project folder, not in the root. We need to remove the root `package.json`